app/views/application/_community_table_index.html.erb

```diff
@@ -12,7 +12,7 @@
   <tbody style= "background-color:white;">
     <% @obituaries.each do |obit| %>
       <tr id="obit-<%= obit.id %>">
-        <td><%= link_to image_tag(obit.image_url, size: "50x50"), "/obituaries/#{obit.id}" %></td>
+        <td><%= link_to image_tag(obit.image_url, width: 75), "/obituaries/#{obit.id}" %></td>
```

Comment:
if we specify a square some photos end up distorted, so switched to specifying the common thumbnail width.